crates/typst/src/export/svg.rs

```diff
@@ -17,6 +18,11 @@ use crate::geom::{
 use crate::image::{Image, ImageFormat, RasterFormat, VectorFormat};
 use crate::util::hash128;
 
+/// The number of segment in a conic gradient.
+/// This is a heuristic value that seems to work well.
+/// Smaller values could be interesting for optimization.
+const CONIC_SEGMENT: usize = 720;
```

Comment:
720 seems like a lot. Is it a lot?

Reply:
Yes it's a lot, but it really depends on the number of stops in the gradient and its size, but since it's SVG we don't know it's size. Ideally you'd do it by scaling the number of stops dynamically with JS (in the webpage I mean). I'll change it to 360 since it's good enough, but this is something that will need more attention in the future.

Probably from someone more knowledgeable about SVG than me.